apps/src/templates/SchoolInfoConfirmationDialog.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {Component, PropTypes} from 'react';
+import Dialog, {Body} from '@cdo/apps/templates/Dialog';
+import Button from './Button';
+import SchoolInfoInterstitial from '../lib/ui/SchoolInfoInterstitial';
+import i18n from '@cdo/locale';
+
+export const styles = {
+  button: {
+    marginTop: 30,
+    marginLeft: 290
+  }
+};
+
+class SchoolInfoConfirmationDialog extends Component {
+  static propTypes = {
+    schoolName: PropTypes.string,
+    onUpdate: PropTypes.func,
+    onConfirm: PropTypes.func,
+    isOpen: React.PropTypes.bool.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showSchoolInterstitial: false,
+      schoolName: props.schoolName
+    };
+  }
+
+  handleUpdateClick = () => {
+    this.setState({showSchoolInterstitial: true});
+  };
+
+  renderInitialContent() {
+    const {onConfirm} = this.props;
+    return (
+      <div style={styles.description}>
+        <Body style={styles.description}>
+          <div>
+            <p>
+              {i18n.schoolInfoDialogDescription()} Lincoln Elementary School?
+            </p>
+          </div>
+        </Body>
+        <Button
+          text={i18n.schoolInfoDialogUpdate()}
+          color={Button.ButtonColor.blue}
+          onClick={this.handleUpdateClick}
+        />
+        <Button
+          style={styles.button}
+          text={i18n.yes()}
+          color={Button.ButtonColor.orange}
+          onClick={onConfirm}
+          href={'www.google.com'}
```

Comment:
Temp link?

Reply:
Yes, temp link is removed in the follow-up email.